src/gui/editors/PianoRoll.cpp

```diff
@@ -3743,6 +3743,12 @@ void PianoRoll::resizeEvent(QResizeEvent* re)
 }
 
 
+void PianoRoll::incrementLeftRightScoll(int value)
+{
+	m_leftRightScroll->setValue(m_leftRightScroll->value() -
+							value * 0.3f / m_zoomXLevels[m_zoomingXModel.value()]);
```

Comment:
Thanks for adding the methods! There's a copy/paste error here. This must be `m_zoomLevels`.

Reply:
Yes, I just realized that. Thanks! (I shouldn't have pushed before I compiled haha)